src/mpid/ch4/shm/include/shm.h

```diff
@@ -18,10 +18,10 @@
 
 /* These typedef function definitions are used when not inlining the shared memory module along
  * with the struct of function pointers below. */
-typedef int (*MPIDI_SHM_mpi_init_hook_t) (int rank, int size, int *n_vcis_provided, int *tag_bits);
+typedef int (*MPIDI_SHM_mpi_init_hook_t) (int rank, int size, int *n_vsis_provided, int *tag_bits);
```

Comment:
Similer to the last comments, interface functions use `vci`; device internally use `vni` or `vsi` where it fits.